tools/android/packaging/Makefile.in

```diff
@@ -102,7 +102,7 @@ python: | xbmc/assets
 	cd xbmc/assets/python2.7/lib/python2.7/; rm -rf test config lib-dynload
 
 res:
-	mkdir -p xbmc/res xbmc/res/raw xbmc/res/values images
+	mkdir -p xbmc/gen/org/xbmc/kodi xbmc/res xbmc/res/raw xbmc/res/values images
```

Comment:
Well, it hurts when it doesn't work. Took me hours to debug https://github.com/koying/SPMC/commit/eaedfe0470e46732cde14d93c72e97e9730260df , also for such parametrization which has not effect on the final product, but KODI_PREFIX was hardcoded somewhere.

So if we can avoid an unnecessary point of failure, let's do it ;)

Besides, org/xbmc/@APP_NAME_LC@ makes no sense, really, as a proper re-branding will have another domain name than xbmc.org. I added @APP_PACKAGE@ to properly handle that.